api/v1beta1/azuremanagedcontrolplane_types.go

```diff
@@ -149,6 +164,9 @@ type AzureManagedControlPlaneSpec struct {
 }
 
 // AADProfile - AAD integration managed by AKS.
+// See also [AKS doc].
+//
+// [AKS doc]: https://docs.microsoft.com/en-us/azure/aks/managed-aad
```

Comment:
```suggestion
// [AKS doc]: https://learn.microsoft.com/azure/aks/managed-aad
```

I think it's preferable to keep `en-us/` out of the URL so the user can land on the appropriate localized doc (if it exists).

Reply:
Makes sense, I caught a few of these in the markdown doc too so I replaced those as well.